src/presenters/abstract-indexable-tag-presenter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Abstract presenter class for indexable presentations.
+ *
+ * @package Yoast\YoastSEO\Presenters
+ */
+
+namespace Yoast\WP\SEO\Presenters;
+
+/**
+ * Abstract_Indexable_Tag_Presenter class
+ */
+abstract class Abstract_Indexable_Tag_Presenter extends Abstract_Indexable_Presenter {
+
+	/**
+	 * The tag format including placeholders.
+	 *
+	 * @var string
+	 */
+	protected $tag_format = '';
+
+	/**
+	 * The method of escaping to use.
+	 *
+	 * @var string
+	 */
+	protected $escaping = 'attribute';
```

Comment:
I think you meant this to do something in `escape` below?

Reply:
Correct, have updated the PR now to indeed use this in the `escape` function as opposed to redeclaring escape for every class.